iphone/Classes/KrollContext.m

```diff
@@ -1141,6 +1143,109 @@ - (int)forceGarbageCollectNow
   return 0;
 }
 
+// Worker class definition
+TiClassRef TiWorker_class(TiContextRef context)
```

Comment:
I'd move all the TiWorker_* stuff over to TiWorkerProxy.m/h. It's specific to that proxy/type.